src/Ssl/CertificateRequest.php

```diff
@@ -24,19 +24,37 @@ class CertificateRequest
     /** @var KeyPair */
     private $keyPair;
 
-    public function __construct(DistinguishedName $distinguishedName, KeyPair $keyPair)
+    /** @var CsrPem */
+    private $csrPem;
+
+    public function __construct(DistinguishedName $distinguishedName, KeyPair $keyPair, string $csr_pem = null)
     {
         $this->distinguishedName = $distinguishedName;
         $this->keyPair = $keyPair;
+        $this->csrPem = $csr_pem;
     }
 
-    public function getDistinguishedName(): DistinguishedName
+    /**
+     * @return DistinguishedName
+     */
+    public function getDistinguishedName()
```

Comment:
You should revert this change

Reply:
I should used git not composer first place. All changes (as are mostly one liners) are made using GitHub web. I'm not a pro PHP programmer, just a hacker ;)
Sorry.